docs/source/science/OPERA/OPERA_Surface_Displacement.ipynb

```diff
@@ -0,0 +1,5744 @@
+{
```

Comment:
<p>The MAAP Pangeo already includes Dask, also when using <code>conda</code> based environments <code>pip</code> is not recommended.</p><p>I think you can make a list of important packages at the top, but don't run <code>pip</code>.  </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/MAAP-Project/maap-documentation/pull/505/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='7'/>